packages/create-webstone-app/src/tasks/2-prepare-project-structure/index.ts

```diff
@@ -2,12 +2,31 @@ import fs from "fs-extra";
 import path, { dirname } from "path";
 import { ListrTask } from "listr2/dist/index";
 import { fileURLToPath } from "url";
+import { temporaryDirectory } from "tempy";
+
+const ignorePaths = [
+  "node_modules",
+  "build",
+  ".svelte-kit",
+  "package",
+  "package-lock.json",
+  "yarn.lock",
```

Comment:
How would we handle this? I think we can take the .gitignore as reference for folders and files that don't matter when copying. The question are lockfiles from package managers: Do we delete them? As we are using the pnpm we can't keep then, but this might cause trouble when installing the project dependencies. What do you think @mikenikles?

Reply:
I think it's fair to say this is no longer an issue once we merge #247, agreed?

Reply:
Yep!